lib/peanut.dart

```diff
@@ -37,7 +37,7 @@ Future<Null> run(
   try {
     var args = ['build', '--output', tempDir.path, targetDir];
 
-    Process process = await Process.start('pub', args);
+    Process process = await Process.start('pub', args, runInShell: true);
```

Comment:
Oops, yeah, `pub.bat`.

No, no issues with this. I was just taking a look through the source - we want a tool like peanut (or peanut itself) for dartdoc.